### PR TITLE
Add example timers

### PR DESCRIPTION
**Pull request by Myzel394**
This PR adds example quick selection timers to the CreateTimerScreen

This allows users to quickly select from common timers, such as 10 minutes, 15 minutes, half an hour, an hour etc.

https://github.com/Bnyro/ClockYou/assets/50424412/efb784d9-c94d-4177-9153-36b1cfa48377

### CODE REVIEW

File: app/src/main/java/com/bnyro/clock/ui/common/ExampleTimers.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            val hours = seconds / 3600;
+            val minutes = (seconds % 3600) / 60;
+            val seconds = seconds % 60;
+
+            return "%02d:%02d:%02d".format(hours, minutes, seconds);
+        }
+}
+
+
+val EXAMPLE_TIMERS = listOf<ExampleTimer>(
```
Comment (Bnyro) [inline app/src/main/java/com/bnyro/clock/ui/common/ExampleTimers.kt:18]: This should be a companion object of example timer.

File: app/src/main/java/com/bnyro/clock/ui/common/ExampleTimers.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ExampleTimer(seconds = 60 * 15),
+    // 20 Minutes
+    ExampleTimer(seconds = 60 * 20),
+    // 30 Minutes
+    ExampleTimer(seconds = 60 * 30),
+    // 60 Minutes
+    ExampleTimer(seconds = 60 * 60),
+    // 90 Minutes
+    ExampleTimer(seconds = 60 * 90),
+)
```
Comment (Bnyro) [inline app/src/main/java/com/bnyro/clock/ui/common/ExampleTimers.kt:37]:
```suggestion
val exampleTimers = listOf(
    60,
    60 * 2,
    ...
).map { ExampleTimer(it) }
```

**Comment by Bnyro** [comment]
Thanks for the PR!
I generally like the idea of these changes, however I think that they do not really fit there in terms of their position in the UI. They look very squeezed there at the bottom.
I think we should either place them above or below the current content (number pad + start button).
Hence the size of the number pad buttons should be reduced as well.

**Comment by Myzel394** [comment]
ncs

File: app/src/main/java/com/bnyro/clock/ui/common/ExampleTimers.kt [hunk 3]
```diff
@@ -12,26 +12,21 @@ data class ExampleTimer(
...
-    // 30 Minutes
-    ExampleTimer(seconds = 60 * 30),
-    // 60 Minutes
-    ExampleTimer(seconds = 60 * 60),
-    // 90 Minutes
-    ExampleTimer(seconds = 60 * 90),
-)
+    companion object {
+        val exampleTimers: List<ExampleTimer>
+            get() = listOf<Int>(
```
Comment (Bnyro) [inline app/src/main/java/com/bnyro/clock/ui/common/ExampleTimers.kt:18]: When using `get()`, the list is regenerated on each call, which we don't want. So just declare them normally.

**Comment by Bnyro** [comment]
In my opinion, we can remove the `:00` at the end of each example timer, because it should be self-explaining that 1s timers don't make any sense.
Otherwise lgtm!

**Comment by Myzel394** [comment]
> In my opinion, we can remove the :00 at the end of each example timer, because it should be self-explaining that 1s timers don't make any sense.

1s do not make sense but 30s would make sense for example. I think most people would interpret "01:00" as one minute and not as one hour

**Comment by Bnyro** [comment]
Agreed, it makes more sense to remove the `00:` at the beginning for anything below one hour.

**Comment by Myzel394** [comment]
I'd prefer to keep them also as this makes all timers the same width and thus easier for eyes to read as one can immediately jump to the minutes and hours parts

**Comment by Bnyro** [comment]
In my opinion that's just wasted space, your screenshot could contain 4 timers instead of the current 3 when removing the 00: at the beginning.

**Comment by Myzel394** [comment]
done

**Review by Bnyro — APPROVED** [review]
Thanks!

**Pull request merged**